src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+}
```

Comment:
Avoid using tag names for styling. It's better to style elements using classes to ensure that the styles are more modular and avoid potential conflicts with other styles.

For example, instead of styling `.stars__star`, if there are other star elements in your project that should not have these styles, consider using a more specific class name.